Content.Server/DeviceLinking/Systems/MemoryCellSystem.cs

```diff
@@ -0,0 +1,74 @@
+using Content.Server.DeviceLinking.Components;
+using Content.Server.DeviceLinking.Events;
+using Content.Server.DeviceNetwork;
+using Content.Shared.DeviceLinking;
+
+namespace Content.Server.DeviceLinking.Systems;
+
+/// <summary>
+/// Handles the control of output based on the input and enable ports.
+/// </summary>
+public sealed class MemoryCellSystem : EntitySystem
+{
+    [Dependency] private readonly DeviceLinkSystem _deviceLink = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<MemoryCellComponent, ComponentInit>(OnInit);
+        SubscribeLocalEvent<MemoryCellComponent, SignalReceivedEvent>(OnSignalReceived);
+    }
+
+    public override void Update(float deltaTime)
+    {
+        base.Update(deltaTime);
+
+        var query = EntityQueryEnumerator<MemoryCellComponent, DeviceLinkSourceComponent>();
+        while (query.MoveNext(out var uid, out var comp, out var source))
+        {
+            if (comp.InputState == SignalState.Momentary)
+                comp.InputState = SignalState.Low;
+            if (comp.EnableState == SignalState.Momentary)
+                comp.EnableState = SignalState.Low;
```

Comment:
Shouldn't a momentary enable trigger an instantaneous update (capturing input -> output)?

Reply:
probably should but not sure how to handle it in UpdateOutput